### PR TITLE
Update get_today function

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
pass in optional date format

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@gsvr please review

File: test/test_utils.js [hunk 1]
```diff
@@ -123,20 +123,28 @@ describe("Testing utils functions", function() {
...
-        });
         it("no date passed, return current moment object", function() {
-            assert.deepEqual(utils.get_today().format("YYYY-MM-DD"),
-                new moment().format("YYYY-MM-DD"));
+            assert.deepEqual(utils.get_today().format(),
+                new moment().format());
+        });
+        it("when date (config) passed in, return corresponding moment object", function() {
+            assert.deepEqual(utils.get_today(config).format(),
+                moment("2016-05-23").format());
```
Comment (gsvr) [inline test/test_utils.js]: Compare to a string please

File: lib/utils.js [hunk 2]
```diff
@@ -168,10 +168,17 @@ get_timestamp = function(format) {
...
+    Defaults to 'YYYY-MM_DD'
 */
-get_today = function(config) {
+get_today = function(config, format) {
     if (config) {
-        return new moment(config.testing_today, 'YYYY-MM-DD');
+        if (format) {
+            return new moment(config.testing_today, format);
+        } else {
+            return new moment(config.testing_today, 'YYYY-MM-DD');
```
Comment (gsvr) [inline lib/utils.js]: rather set up the value for `format` to default to before calling it

**Comment by NicolaasDieBaas** [comment]
@gsvr please review again

File: test/test_utils.js [hunk 3]
```diff
@@ -123,20 +123,25 @@ describe("Testing utils functions", function() {
...
+        });
+        it("when date (config) passed in, return corresponding moment object", function() {
+            assert.deepEqual(utils.get_today(config).format("YYYY-MM-DD"), "2016-05-23");
+        });
+        it("when date (config) & format passed in, return corresponding moment object", function() {
+            assert.deepEqual(utils.get_today(config, "YYYY-MM-DD hh:mm:ss").format("YYYY-MM-DD hh:mm:ss"),
+                "2016-05-23 12:30:15");
+        });
+        it("when date (config) & format passed in, evaluates to false because of difference in time", function() {
+            assert.deepEqual(utils.get_today(config, "YYYY-MM-DD hh:mm:ss").format("YYYY-MM-DD hh:mm:ss")
```
Comment (gsvr) [inline test/test_utils.js]: Rather use `assert.notEqual` (whatever the right syntax is).

**Comment by NicolaasDieBaas** [comment]
@gsvr all ok now?

**Pull request merged**